quiffen/core/transaction.py

```diff
@@ -396,7 +397,7 @@ def from_list(
                         field_info.split(" ")[0]
                     )
             elif line_code in {"T", "U"}:
-                amount = utils.parse_decimal(field_info, locale=locale)
+                amount = parse_decimal(field_info, locale)
```

Comment:
The `utils.parse_decimal` function is used in many other places. It might be better to just change the function in `utils`, and make sure the `locale` is passed to all callsites.

Also, it might be worth updating docs to show that `locale` is an option.